### PR TITLE
Issue #359, flag for output additional properties for Error object

### PR DESCRIPTION
**Pull request by ilyaztsv**
I've added `errorProps` flag which allow to output additional properties for json of Error object.

### CODE REVIEW

**Comment by mcollina** [comment]
Can you please add a screenshot of the result after merging this? I think we might even kill the option: showing these properties is important all the time.

cc @jsumners

**Comment by ilyaztsv** [comment]
@mcollina do you mean screenshot of the output if I use new flag?

**Comment by mcollina** [comment]
yes, thanks!

**Comment by ilyaztsv** [comment]
default-error-output
error-output-with-errorprops-flag

File: docs/cli.md [hunk 1]
```diff
@@ -74,3 +74,34 @@ Into this:
 ```sh
 [2017-04-25T17:32:09.662Z] [32mINFO[39m (24280 on SP2): [36mhello world[39m
 ```
+
+If you logged instance of Error object, Pino adds `"type":"Error"` to json.
```
Comment (jsumners) [inline docs/cli.md]: => "If an instance of `Error` is logged, Pino adds `"type":"Error"` to the logged JSON.

File: docs/cli.md [hunk 2]
```diff
@@ -74,3 +74,34 @@ Into this:
...
+By default, Pino will transform this:
```
Comment (jsumners) [inline docs/cli.md]: > "Thus, when prettifying the output, Pino will transform the JSON:"

File: docs/cli.md [hunk 3]
```diff
@@ -74,3 +74,34 @@ Into this:
...
+
+```js
+{"level":50,"time":1457537229339,"msg":"Error message.","pid":44127,"hostname":"MacBook-Pro-3.home","type":"Error","stack":"Stack of the error","statusCode":500,"v":1}
+```
+
+Into this:
```
Comment (jsumners) [inline docs/cli.md]: => "To:"

File: docs/cli.md [hunk 4]
```diff
@@ -74,3 +74,34 @@ Into this:
...
+
+```sh
+ERROR [2016-03-09T15:27:09.339Z] (44127 on MacBook-Pro-3.home): Error message.
+    Stack of the error
+```
+
+If you want to output additional properties for json of Error object, you can use `--errorProps <properties>` flag.
```
Comment (jsumners) [inline docs/cli.md]: => "To log additional properties of `Error` objects, supply the `--errorProps <properties>` flag."

File: docs/cli.md [hunk 5]
```diff
@@ -74,3 +74,34 @@ Into this:
...
+
+`pino --errorProps statusCode,v` will transform this:
```
Comment (jsumners) [inline docs/cli.md]: => "For example, `pino --errorProps statusCode,v` will transform:"

File: docs/cli.md [hunk 6]
```diff
@@ -74,3 +74,34 @@ Into this:
...
+
+```js
+{"level":50,"time":1457537229339,"msg":"Error message.","pid":44127,"hostname":"MacBook-Pro-3.home","type":"Error","stack":"Stack of the error","statusCode":500,"v":1}
+```
+
+Into this:
```
Comment (jsumners) [inline docs/cli.md]: => "To:"

File: docs/cli.md (hunk 1 above)
Comment (ilyaztsv) [reply docs/cli.md]: fixed

File: docs/cli.md (hunk 2 above)
Comment (ilyaztsv) [reply docs/cli.md]: fixed

File: docs/cli.md (hunk 3 above)
Comment (ilyaztsv) [reply docs/cli.md]: fixed

File: docs/cli.md (hunk 4 above)
Comment (ilyaztsv) [reply docs/cli.md]: fixed

File: docs/cli.md (hunk 5 above)
Comment (ilyaztsv) [reply docs/cli.md]: fixed

File: docs/cli.md (hunk 6 above)
Comment (ilyaztsv) [reply docs/cli.md]: fixed

**Pull request merged**